app/src/main/res/layout/list_home_item.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Same here. list_home_item does not say to much to me. What kind of item is it?
That's why we always choose a descriptive name, related to what we want to display or manage: subscriter_item.xml or similar could work better.